app/views/user_mailer/reset_link.html.erb

```diff
@@ -0,0 +1,14 @@
+<div>
+  <p>You're receiving this e-mail because you or someone else has requested a password reset for your SSID account.</p>
+
+  <p>
+    Click the link below to reset your password:<br>
+    <%= @reset_link %>
+  </p>
+  
+  <p>If you did not request a password reset you can safely ignore this email.</p>
+  
+  <p>Thank you.</p>
+  
+  <p>The SSID Team</p>
```

Comment:
Use constants instead of strings? "The SSID Team" => SSID_Maintainer

Reply:
Same as above.